orders/views.py

```diff
@@ -86,10 +99,99 @@ def post(self, request):
                 item                            = Item.objects.get(id=order_item.item_id)
                 item.order_quantity             += order_item.quantity
                 item.stock                      -= order_item.quantity               
+                
+                if Cart.objects.filter(Q(member_id=request.user.id) & Q(item_id = order_item.item_id)).exists():
+                    Cart.objects.filter(Q(member_id=request.user.id) & Q(item_id = order_item.item_id)).delete()
+
                 order_item.save()
                 item.save()
 
             return JsonResponse({'MESSAGE': "SUCCESS"}, status=201)
         
         except KeyError:
-            return JsonResponse({"MESSAGE": "KEY_ERROR"}, status=400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE": "KEY_ERROR"}, status=400)
+
+
+
+
+class CartView(View):
+    @login
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if Cart.objects.filter(Q(item_id = data['item_id']) & Q(member_id=request.user.id)).exists():
+                return JsonResponse({"MESSAGE":"CART_EXIST"}, status=400)
```

Comment:
중복되는 경우는 `409`를 반환합니다.

```suggestion
            if Cart.objects.filter(Q(item_id = data['item_id']) & Q(member_id=request.user.id)).exists():
                return JsonResponse({"MESSAGE":"CART_ALEADY_EXIST"}, status=409)
```